packages/grafana-data/src/dataframe/processDataFrame.ts

```diff
@@ -241,6 +270,11 @@ export const toDataFrame = (data: any): DataFrame => {
     return new MutableDataFrame(data as DataFrameDTO);
   }
 
+  // Handle legacy docs/json type
+  if (data.hasOwnProperty('type') && data.type === 'docs') {
+    return convertJSONDocumentDataToDataFrame(data);
+  }
+
```

Comment:
Is the only difference between this and `convertTimeSeriesToDataFrame` that it adds:
```
    meta: { json: true },
```
If so, do we need the new function?  or just add a note about deprecated/legacy docs mode?

Reply:
No, `convertTimeSeriesToDataFrame` expects a time and value besides some other differences. See https://github.com/grafana/grafana/pull/19596/files#diff-0600e6a1b61c8024cc2e5cc95d22c54bR130-R156